Clinical-Genomics/schug
# Fixed missing header in files downloaded from Ensembl - and deprecated pkg_resources

### This PR adds | fixes:
- Fix #19 
- Fix #20 
- Adds headers to the demo files

### How to test:
- Download one of the Ensembl files and make sure it has an header
- All automatic tests should pass -> This will test the pkg_resources fix

### Review:

This [version](https://semver.org/) is a:
